networking/network_policy/about-network-policy.adoc

```diff
@@ -5,11 +5,666 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-As a cluster administrator, you can define network policies that restrict traffic to pods in your cluster.
+As a project administrator, you can define network policies that restrict traffic to or from pods in a namespace.
```

Comment:
Not sure the "in a namespace" qualifier is correct here; NetworkPolicy isn't inherently namespace specific.